### PR TITLE
Add support for secrets loading from SECRET_ARN

### PR DESCRIPTION
**Pull request by seanspeaks**
AWS specific that could be modified to be independent

Will not attempt if no SECRET_ARN is set in ENVs

### CODE REVIEW

**Comment by seanspeaks** [comment]
Task linked: [FRI-460 Move SecretsToEnv to frigg core, add to createHandler](https://app.clickup.com/t/9013279251/FRI-460)

File: packages/core/core/create-handler.js [hunk 1]
```diff
@@ -20,6 +21,15 @@ const createHandler = (optionByName = {}) => {
         try {
             initDebugLog(eventName, event);
 
+            const requestMethod = event.httpMethod;
+            const requestPath = event.path;
+            if (requestMethod && requestPath) {
```
Comment (d-klotz) [inline packages/core/core/create-handler.js:26]: what's the use of this variables besides the loging?

Comment (seanspeaks) [reply packages/core/core/create-handler.js:26]: Just logging

(Thought it was useful context in another project... could remove or really deserves a full revamp of logging/telemetry across Frigg)

File: packages/core/core/secrets-to-env.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+const getSecretValue = async () => {
+    console.log('Fetching secrets...');
+
+    const httpPort = process.env.PARAMETERS_SECRETS_EXTENSION_HTTP_PORT || 2773;
```
Comment (d-klotz) [inline packages/core/core/secrets-to-env.js:4]: Should we really default here? If this is a env var used in more places, we should throw an error if it's not provided.

**Review by d-klotz — APPROVED** [review]

**Comment by seanspeaks** [comment]
:rocket: PR was released in [`v1.2.2`](https://github.com/friggframework/frigg/releases/tag/v1.2.2) :rocket:

**Pull request merged**